### ISSUE TITLE
SQLwatch on RDS

### ISSUE THREAD

**Issue by tune007** [marcingminski/sql-performance-monitor#483]
**Did you check DOCS to make sure there is no workaround?**
https://sqlwatch.io/docs/
The Docs does not make specific mention of deploying to RDS. Can you please give me some guidance on how I can make SQLwatch work on RDS, I would love to help with the testing or assist with anything related to get this working as far as possible from my side.

**Describe the bug**
There are several issues when deploying to and RDS instance because there are no sysadmin permissions like for instance creating an assembly on the master database
There is no option to create a proxy in RDS to enable a user to run PowerShell jobs
Manually deployed extended events because the default directory for extended events in RDS is different from the dacpac

**To Reproduce**
Occurs during testing of SQLWatch job execution and when trying to deploy dacpac

**Expected behavior**
SQLwatch deployment fails with various errors but starting with an issue deploying to specific directory:
Error SQL72014: Core Microsoft SqlClient Data Provider: Msg 50000, Level 16, State 1, Procedure rds_extended_events_trigger, Line 172 Unable to create/alter extended events session: Extended events event_file target is allowed only to D:\rdsdbdata\Log
Error SQL72045: Script execution error. The executed script:
CREATE EVENT SESSION [SQLWATCH_blockers] ON SERVER

**Screenshots**

**Windows Server (please complete the following information):**
 - OS Version: [e.g. Windows 2012 R2]

**SQL Server (please complete the following information):**
 - SQL Version: [e.g. SQL Server 2016 SP1]
 - SQL Edition: [e.g. Enterprise]
 
**SQL Server Management Studio (SSMS -> about -> copy info):**
 - paste "about" info here:
 
**SQLWATCH version (from DACPAC or from sysinstances)**
SQLWATCH 4.8.0.548 20230409190820
 
**Additional context**

**Comment by marcingminski** [comment]
Is this a duplicate of 
https://github.com/marcingminski/sqlwatch/issues/375 ?

**Comment by tune007** [comment]
not really, my issue stems from the fact that sysadmin is not allowed and therefore I cannot run the PowerShell jobs, there is also not an option for a proxies available in RDS. We are migrating our workload to AWS and I am trying to establish if it is worth my while to pursue using SQLwatch as it has been working well for tests done for on prem

**Comment by marcingminski** [comment]
Would you be interested in the Cloud version, im looking for businesses with AWS workload to make it compatible with RDS.
This may then eventually trickle into the open source.

**Comment by tune007** [comment]
Hi Marcin, possibly. Please share your email then we can chat "offline" about this.

**Comment by marcingminski** [comment]
Would you like to go to https://sqlwatch.io/cloud/ and book a slot for a Zoom chat - so I can show you how it works and discuss details. Don't worry, its not going to be a sales pitch.
